docs/how-to-build-with-ddn/with-snowflake.mdx

```diff
@@ -0,0 +1,307 @@
+---
+sidebar_position: 6
+sidebar_label: With Snowflake
+description: "Learn the basics of Hasura DDN and how to get started with a Snowflake instance."
+keywords:
+  - hasura ddn
+  - graphql api
+  - getting started
+  - guide
+  - snowflake
+---
+
+import Prereqs from "@site/docs/_prereqs.mdx";
+
+# Get Started with Hasura DDN and Snowflake
+
+## Overview
+
+This tutorial takes about twenty minutes to complete. You'll learn how to:
+
+- Set up a new Hasura DDN project
+- Connect it to a Snowflake instance
+- Generate Hasura metadata
+- Create a build
+- Run your first query
+
+Additionally, we'll familiarize you with the steps and workflows necessary to iterate on your API.
+
+This tutorial assumes you're starting from scratch, but you can easily follow the steps if you already have data seeded;
+Hasura will never modify your source schema.
+
+<Prereqs />
+
+## Tutorial
+
+### Step 1. Authenticate your CLI
+
+```sh title="Before you can create a new Hasura DDN project, you need to authenticate your CLI:"
+ddn auth login
+```
+
+This will launch a browser window prompting you to log in or sign up for Hasura DDN. After you log in, the CLI will
+acknowledge your login, giving you access to Hasura Cloud resources.
+
+### Step 2. Scaffold out a new local project
+
+```sh title="Next, create a new local project:"
+ddn supergraph init my-project && cd my-project
+```
+
+Once you move into this directory, you'll see your project scaffolded out for you. You can view the structure by either
+running `ls` in your terminal, or by opening the directory in your preferred editor.
+
+### Step 3. Initialize your Snowflake connector
+
+```sh title="In your project directory, run:"
```

Comment:
Can we clarify what the user needs to put in the connector interactive mode for the schema array answer or if they can leave that out if they specify it in the connection string.

Reply:
This is something we should consider for all the tutorials; as I write that, since this _is_ a tutorial, I'm comfortable being prescriptive here and would prefer not to cloud the steps with deviations.

Reply:
I would say I was very unsure when going though it what to enter at that step and what difference it would make to the connector. Especially since there is no validation of whether it's done correctly during this command. User was confused.

Reply:
Got it; I'll make a follow-up 👍